### PR TITLE
Add option to lazy load cards

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Don't include less-used cards in the main Lovelace bundle, instead load them on demand.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/create-element/create-element-base.ts [hunk 1]
```diff
@@ -50,14 +50,40 @@ const _createErrorElement = <T extends keyof CreateElementConfigTypes>(
...
+  tag: string,
+  config: CreateElementConfigTypes[T]["config"]
+) => {
+  if (customElements.get(tag)) {
+    return _createElement(tag, config);
+  }
+
+  const element = _createErrorElement(
+    `Custom element doesn't exist: ${tag}.`,
+    config
```
Comment (bramkragten) [inline src/panels/lovelace/create-element/create-element-base.ts]: This should be the same error as here: https://github.com/home-assistant/home-assistant-polymer/pull/4857/files#diff-15d91f0ebd111fc153e36f517963b336R134

Comment (balloob) [reply src/panels/lovelace/create-element/create-element-base.ts]: Your link doesn't work anymore.

**Comment by thomasloven** [comment]
This will break some custom cards in ways that cannot be solved without #4853.

**Comment by bramkragten** [comment]
> This will break some custom cards in ways that cannot be solved without #4853.

Yes, we are aware of that.

**Comment by balloob** [comment]
@thomasloven examples please. Using the new card helpers you should be able to get anything imported that this PR marks as dynamic import.

**Comment by thomasloven** [comment]
I assumed you were, but like to have things in writing for when people stumble on this in a blame or so.
@balloob I said they cannot be solved *without* 4853. With that, it's a great change. But that should block this one IMO.

**Comment by balloob** [comment]
Oh my bad, I misread. Yes, you are right this needs 4853.

**Pull request merged**